tests/common/runners/test_object_runner.py

```diff
@@ -0,0 +1,848 @@
+         (6, '# You may wish to alter this file to override the set of languages analyzed,\n'),
+         (7, '# or to provide custom queries or build logic.\n'), (8, '#\n'), (9, '# ******** NOTE ********\n'),
+         (10, '# We have attempted to detect the languages in your repository. Please check\n'),
+         (11, '# the `language` matrix defined below to confirm you have the correct set of\n'),
+         (12, '# supported CodeQL languages.\n'), (13, '#\n'), (14, 'name: "CodeQL"\n'), (15, '\n'), (16, 'on:\n'),
+         (17, '  push:\n'), (18, '    branches: [ master ]\n'), (19, '  pull_request:\n'),
+         (20, '    # The branches below must be a subset of the branches above\n'), (21, '    branches: [ master ]\n'),
+         (22, '  schedule:\n'), (23, "    - cron: '17 4 * * 2'\n"), (24, '  workflow_dispatch:\n'), (25, '\n'),
+         (26, '\n'), (27, 'jobs:\n'), (28, '  analyze:\n'), (29, '    name: Analyze\n'),
+         (30, '    runs-on: [self-hosted, public, linux, x64]\n'), (31, '    permissions:\n'),
+         (32, '      actions: read\n'), (33, '      contents: read\n'), (34, '      security-events: write\n'),
+         (35, '    steps:\n'), (36, '      - name: Checkout repository\n'), (37, '        uses: actions/checkout@v3\n'),
+         (38, '      - name: Set up Python\n'), (39, '        uses: actions/setup-python@v4\n'),
+         (40, '        with:\n'), (41, "          python-version: '3.10'\n"),
+         (42, '      - name: Setup python for CodeQL\n'), (43, '        run: |\n'),
+         (44, '          python -m pip install --upgrade pip pipenv\n'),
+         (45, '          echo "CODEQL_PYTHON=$(which python)" >> $GITHUB_ENV\n'),
+         (46, '      - name: Check Pipfile.lock changed\n'),
+         (47, '        uses: tj-actions/verify-changed-files@v10\n'), (48, '        id: changed_files\n'),
+         (49, '        with:\n'), (50, '          files: Pipfile.lock\n'),
+         (51, '      - name: Setup dependencies if they changed\n'),
+         (52, "        if: steps.changed_files.outputs.files_changed == 'true'\n"), (53, '        run: |\n'),
+         (54, '          pipenv lock -r > requirements.txt\n'), (55, '          pip install -r requirements.txt\n'),
+         (56, '      - name: Initialize CodeQL\n'), (57, '        uses: github/codeql-action/init@v2\n'),
+         (58, '        with:\n'), (59, '          languages: python\n'),
+         (60, '          setup-python-dependencies: false\n'), (61, '      - name: Autobuild\n'),
+         (62, '        uses: github/codeql-action/autobuild@v2\n'), (63, '      - name: Perform CodeQL Analysis\n'),
+         (64, '        uses: github/codeql-action/analyze@v2\n')])
+
+    def test_get_jobs_on_result(self):
+        expected_jobs = {"bridgecrew": {"__startline__": 8, "__endline__": 20},
+                         "bridgecrew2": {"__startline__": 21, "__endline__": 33}, }
+
+        jobs = Runner._get_jobs(self, self.result[0])
```

Comment:
@NoaAzoulay This is testing the GHA runner, not the object runner. Why put this test here?